src/styles/global.scss

```diff
@@ -0,0 +1,2 @@
+@import 'fonts';
```

Comment:
It looks like these should be double quotes -- I'm assuming this is from `stylelint.config.js`? Are double quotes the standard for CSS? Or should we change the rule in `stylelint.config.js` to single, to match the JS rule? (Ditto for the other `.scss` files.)